.velocitas.json

```diff
@@ -0,0 +1,31 @@
+{
+    "packages": [
+        {
+            "name": "devenv-runtime-local",
+            "version": "v1.0.0"
+        },
+        {
+            "name": "devenv-runtime-k3d",
+            "version": "v1.0.0"
+        },
+        {
+            "name": "devenv-github-workflows",
+            "version": "cpp-template"
+        },
+        {
+            "name": "devenv-github-templates",
+            "version": "v1.0.0"
+        },
+        {
+            "name": "devenv-devcontainer-setup",
+            "version": "cpp-template"
```

Comment:
Could that stay this way or do we need to release the changes in that repo first?

Reply:
We need to release the changes in that repo first, I will provide a PR.

Reply:
Fixed